source/search.html

```diff
@@ -2174,51 +2253,344 @@ <h4>Prefixes</h4>
  </tr>
  <tr>
 	<td><code>sa</code></td>
-	<td>the resource value starts after the parameter value</td>
-  <td>the range of the parameter value does not overlap with the range of the resource value, and the range above the parameter value contains the range of the resource value</td>
+	<td>The resource value starts after the parameter value.</td>
+  <td>The range of the parameter value does not overlap with the range of the resource value, and the range above the parameter value contains the range of the resource value.
+      <br/><b>Boundary interpretation:</b> The lower boundary of the resource value is greater than the upper boundary of the parameter value.
+      <br/><b>Resource interval tests:</b>
+      <br/>&nbsp;<code>resource-lower: (parameter-upper, ∞)</code>
+  </td>
  </tr>
  <tr>
 	<td><code>eb</code></td>
-	<td>the resource value ends before the parameter value</td>
-  <td>the range of the parameter value does not overlap with the range of the resource value, and the range below the parameter value contains the range of the resource value</td>
+	<td>The resource value ends before the parameter value.</td>
+  <td>The range of the parameter value does not overlap with the range of the resource value, and the range below the parameter value contains the range of the resource value.
+      <br/><b>Boundary interpretation:</b> The upper boundary of the resource value is less than the lower boundary of the parameter value.
+      <br/><b>Resource interval tests:</b>
+      <br/>&nbsp;<code>resource-upper: (-∞, parameter-lower)</code>
+  </td>
  </tr>
  <tr>
 	<td><code>ap</code></td>
-	<td>the resource value is approximately the same to the parameter value.<br/>
-	Note that the recommended value for the approximation is 10% of the stated value (or for a date, 10% of the gap between now and the date), but systems may choose other values where appropriate</td>
-  <td>the range of the parameter value overlaps with the range of the resource value</td>
+	<td>The resource value is approximately the same to the parameter value.
+      <br/> Note that the recommended value for the approximation is 10% of the stated value (or for a date, 10% of the gap between now and the date), but systems may choose other values where appropriate.</td>
+  <td>The range of the parameter value overlaps with the range of the resource value.
+      <br/><b>Boundary interpretation:</b> The lower boundary of the resource value is less than or approximately the upper boundary of the parameter value
+       AND the upper boundary of the resource value is greater than or approximately the lower boundary of the parameter value.
+      <br/>&nbsp;<code>resource-lower: [~parameter-lower, ~parameter-lower]</code>
+      <br/>&nbsp;<code>resource-upper: [~parameter-upper, ~parameter-upper]</code>
+  </td>
  </tr>
 </table>
+
+<p>
+  Following is a table showing some example ranges and how they interact with the prefixes, with <code>[P]</code> representing the parameter

```

Comment:
Again, should this be marked informative?

Reply:
I am fine with marking it informative or not - anyone else have thoughts?

Fixed the extra-line formatting. If the images are clearer, we can swap over - again, anyone else have a preference?